### PR TITLE
Action Policy updates

### PR DESCRIPTION
**Pull request by EricDahlvang**
Note: this should be cherry-picked to main (or, merged to main and cherry-picked to 4.12)

PR for policy definitions in schema here:
https://github.com/microsoft/botframework-sdk/pull/6208/files

Thanks @chrimc62 !

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BreakLoop.schema [hunk 1]
```diff
@@ -20,9 +20,9 @@
             ]
         }
     },
-    "policies": [
+    "$policies": [
         {
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BreakLoop.schema:24]: All of these are invalid.  It is not an array.  If you open these in VS code you will see errors.  It should be "$policies": { lastAction: true }

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnEndOfConversationActivity.schema [hunk 2]
```diff
@@ -6,9 +6,9 @@
     "type": "object",
     "required": [
     ],
-    "policies": [
+    "$policies": [
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnEndOfConversationActivity.schema:9]: >"$policies":  [](start = 4, length = 13)

Should be $policies: { notInteractive: true}

**Review by chrimc62 — CHANGES_REQUESTED** [review]
:clock1:

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BreakLoop.schema (hunk 1 above)
Comment (EricDahlvang) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BreakLoop.schema]: oops :/

Thanks @chrimc62 !

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnEndOfConversationActivity.schema (hunk 2 above)
Comment (EricDahlvang) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnEndOfConversationActivity.schema]: #fixed

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Pull request merged**